### ISSUE TITLE
Find potential differences in rules triggered between sandbox and local installation

### ISSUE THREAD

**Issue by theseion** [coreruleset/coreruleset#2750]

**Comment by dune73** [comment]
Also see https://github.com/coreruleset/crs-sandbox/issues/48

**Comment by RedXanadu** [comment]
Is the issue that "nightly" on Apache is not up to date? That would be a nice and easy answer :smile: 

I spoke briefly to @theMiddleBlue about this the other week: they offered to update prod, but I said "it's fine, I'll just test locally" :sweat_smile: 

Do we need an automated way to update the sandbox machines / rule sets?

**Comment by dune73** [comment]
I doubt this is an "outdated sandbox" problem. But let's first draw a clear picture first and the root cause might be pretty clear afterwards.

**Comment by dune73** [comment]
@theMiddleBlue, I think you worked on this, did not you? Also a discussion in Varese.

Really no longer sure what the status is here. Can you help out?

**Comment by theMiddleBlue** [comment]
yes, it's part of the work I'm doing by running all our regression tests against the sandbox. It's a huge work because I'm categorizing every exception. Spoiler: basically, for now, all failing tests fail because Nginx doesn't allow certain requests or the sandbox requires a specific configuration that prevents matching some payload.

**Comment by dune73** [comment]
Thank you.

So the work is so huge because there are indeed a lot of exceptions?

Can you quantify in percentage of overall tests? If it's substantial, we should explain this on the sandbox page in /docs.

**Comment by EsadCetiner** [comment]
Is there still a divergence between the behavior of the sandbox and a local install? From what I've tested I can't find any divergence.

**Comment by theseion** [comment]
Let's close this one. The sandbox setup was rewritten.
